404.html

```diff
@@ -0,0 +1,191 @@
+      rel="stylesheet"
+    />
+    <title>404 - page not found</title>
+    <style>
+      .content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        padding: 80px 0 80px 0;
+      }
+
+      .heading {
+        font-size: 64px;
+        font-weight: bold;
+      }
+
+      .subheading {
+        font-size: 32px;
+        font-weight: bold;
+      }
+
+      .centered {
+        margin: 50px 0;
+      }
+
+      .back-to-home {
+        text-align: center;
+        margin-top: 40px;
+      }
+
+      .button {
+        padding: 10px 20px;
+        background-color: #f28138;
+        color: white;
+        text-decoration: none;
+        border-radius: 5px;
+        transition: background-color 0.3s ease;
+        font-weight: 600;
+      }
+
+      .button:hover {
+        background-color: #ea580c;
+      }
+      footer {
+        height: 100vh; /* Set the height to 100% of the viewport height */
+      }
+    </style>
+  </head>
+  <body class="font-sans">
+    <header>
+      <nav class="flex items-center justify-between py-2 px-4">
+        <div>
+          <a href="/index.html">
+            <img
+              src="/assets/logo/travelup.svg"
+              alt="TravelUp logo"
+              width="169"
+              height="100"
+              class="py-2"
+            />
+          </a>
+        </div>
+        <div class="flex space-x-5">
+          <a
+            href="/index.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Home</a
+          >
+          <a
+            href="/about.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >About</a
+          >
+          <a
+            href="/shop.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Shop</a
+          >
+          <a
+            href="/blog.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Blog</a
+          >
+          <a
+            href="/contact.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Contact</a
+          >
+        </div>
+      </nav>
+    </header>
+    <main>
+      <div>
+        <div class="content">
+          <div class="centered">
+            <img src="assets/logo/travelup.svg" alt="" />
+          </div>
+          <div class="centered">
+            <h1 class="heading">404</h1>
+          </div>
+          <div class="centered">
+            <h2 class="subheading">Page not found</h2>
+          </div>
+          <div class="back-to-home">
+            <a href="/" class="button">Back to Home</a>
+          </div>
```

Comment:
Do you think you could use the <button> tag for this within the <a> tag?
Also needs to link back to the home page.

Reply:
Completed Hamster H